lms/djangoapps/courseware/views/index.py

```diff
@@ -436,6 +442,10 @@ def _create_courseware_context(self):
                 table_of_contents['previous_of_active_section'],
                 table_of_contents['next_of_active_section'],
             )
+            # sections can hide data that masquerading staff should see when debugging issues with specific students
+            section_context['specific_masquerade'] = (
```

Comment:
This code should be inside the `_create_section_context` method instead, since it's logically for creation of the section context.